### ISSUE TITLE
On HA 0.88.0 does the component not working anymore

### ISSUE THREAD

**Issue by noam148** [vrih/homeassistant-rn301#2]
On HA 0.88.0 does the component not working anymore. I replaced some code and it work again. If you need i can do a pull request. Let me know if you need it or you wanna fix it by your self.

**Comment by cristimi** [comment]
Hi, I am using the code as well with some tweaks on the volume.  I am currently on HA 0.87.1 but planning to update next week to 0.88. Is it possible to have the updated code?

**Comment by noam148** [comment]
Yes, of course. I fixed the code by checking the errors from the HA log and check the code by other components. @vrih Please check the code #3  before merge, because i don't work alot with python.
